src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,96 @@
+    ```
+
+1. Deploy the prover:
```

Comment:
```suggestion
1. Deploy the multisig prover. You can also instantiate the [existing](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json#L434) multisig prover contract which has code id `8` instead of deploying a new one:

```